editor/import/editor_import_plugin.cpp

```diff
@@ -112,6 +112,14 @@ int EditorImportPlugin::get_import_order() const {
 	ERR_FAIL_V_MSG(-1, "Unimplemented _get_import_order in add-on.");
 }
 
+int EditorImportPlugin::get_format_version() const {
+	int ret;
```

Comment:
```suggestion
	int ret = 0;
```
Or other appropriate value

Reply:
Thanks! Changed in latest push